src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -313,6 +313,20 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
         ssl_cert=config['sasTestHarnessCert'],
         ssl_key=config['sasTestHarnessKey'])
 
+    # Write SAS UUT PPA to output directory of PCR.1 test.
+    # PPA Zone received from SAS UUT in PCR.1 test will be considered as input
+    # for PCR 3,PCR 6 and PCR 7 tests.
+    ppa_zone_data_dir_path = os.path.join('testcases', 'output', 'test_WINNF_FT_S_PCR_1')
+    ppa_zone_data_file_name = 'sas_uut_claimed_ppa_boundary_' + \
```

Comment:
Consider pulling this out into its own function so that PCR.1 and PCR.3 do the same thing.

Reply:
Moved   these section to getSasUutClaimedPpaBoundaryFilePath function to return the   absolute path of PPA zone is stored that is received from SAS UUT as part of   PCR 1.